ItemList/Source/ItemList.Contracts/Bootstrap/IBootstrap.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ItemList.Contracts.Bootstrap
```

Comment:
You actually call it `.DependencyInjection` now. It is somewhat misleading having `.DependencyInjection` implement `.Bootstrap` contract. Consider renaming the namespace.